src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/custom.py

```diff
@@ -30,43 +30,13 @@ def wrong_vmsize_argument_exception_handler(ex):
 def cli_redis_export(cmd, client, resource_group_name, name, prefix, container, file_format=None):
     from azure.mgmt.redis.models import ExportRDBParameters
     parameters = ExportRDBParameters(prefix, container, file_format)
-    return client.export(resource_group_name, name, parameters)
-
-
-# pylint: disable=unused-argument
-def cli_redis_import_method(cmd, client, resource_group_name, name, file_format, files):
-    from azure.mgmt.redis.models import ImportRDBParameters
-    parameters = ImportRDBParameters(files, file_format)
-    return client.import_method(resource_group_name, name, files, parameters)
+    return client.export_data(resource_group_name, name, parameters)
 
 
 # pylint: disable=unused-argument
-def cli_redis_list(cmd, client, resource_group_name=None):
-    if resource_group_name:
-        return client.list_by_resource_group(resource_group_name)
-    return client.list()
-
-
-# pylint: disable=unused-argument
-def cli_redis_update_settings(cmd, client, resource_group_name, name, redis_configuration):
-    from azure.mgmt.redis.models import RedisUpdateParameters
-
-    existing = client.get(resource_group_name, name)
-    existing.redis_configuration.update(redis_configuration)
-
-    # Due to swagger/mgmt SDK quirkiness, we have to manually copy over
-    # the resource retrieved to a create_parameters object
-    update_params = RedisUpdateParameters(  # pylint: disable=too-many-function-args
-        existing.redis_configuration,
-        existing.enable_non_ssl_port,
-        existing.tenant_settings,
-        existing.shard_count,
-        existing.subnet_id,
-        existing.static_ip,
-        existing.sku,
-        existing.tags
-    )
-    return client.update(resource_group_name, name, parameters=update_params)
+def cli_redis_import_method(cmd, client, resource_group_name, name, files, file_format=None):
+    logger.warning('This command is getting deprecated. Please use "redis import" command')
```

Comment:
This warning should be unnecessary if you used deprecate when registering the command.